tests/e2e/mapper/bluetooth/bluetooth_test.go

```diff
@@ -108,175 +97,14 @@ func ScheduleExecute(client MQTT.Client, message MQTT.Message) {
 // Checks whether mapper has written data correctly to connected device
 var _ = Describe("Application deployment test in E2E scenario", func() {
 	Context("Test write characteristic of mapper", func() {
-		BeforeEach(func() {
```

Comment:
Does it mean that remove those tests?

Reply:
These tests all need refactor, I cam comment them instead of deleting.

Reply:
These tests all need refactor, I cam comment them instead of deleting.